search-app/docs/api.md

```diff
@@ -235,6 +235,7 @@ These are query parameters that control what extra data is included with collect
   * `include_has_granules` - If this parameter is set to "true" this will include a flag indicating true or false if the collection has any granules at all. Supported in all response formats except opendata. To limit search results to collections with or without granules, see the [`has_granules`](#c-has-granules) parameter.
   * `include_granule_counts` - If this parameter is set to "true" this will include a count of the granules in each collection that would match the spatial and temporal conditions from the collection query. Supported in all response formats except opendata.
   * `include_facets` There are 3 values allowed: "true", "false", or "v2". If this parameter is set to "true" or "v2" facets will be included in the collection results (not applicable to opendata results). Facets are described in detail below.
+  * `facets_size[facet]` is used to customize the max number of values displayed for the facet. Values allowed: positive integer. Supported facets are: science-keywords, platform, instrument, data-center, project, processing-level-id and variables.  
```

Comment:
Change `facets_size[facet]` to something like `facets_size[field-name]` and update the doc to make clear the usage by adding an example.

Reply:
Okay I did add an example in the section below.